immuni_exposure_ingestion/tasks/process_uploads.py

```diff
@@ -60,70 +61,145 @@ async def _process_uploads() -> None:
     # Acquire a lock on redis before processing anything, avoiding concurrent tasks.
     async with lock_concurrency("process_uploads"):
         _LOGGER.info("Obtained lock.")
+        batch_it()
+        batch_eu()
```

Comment:
```suggestion
        _batch_it()
        _batch_eu()
```

Reply:
Okay, I see that in the other task you're processing the EU ones separately.
Thus, I'd first need you to better explain what's the goal so as to provide a better review. 🙏

Reply:
yes, you're right. Let me clarify:
_batch_eu() (private method as suggested) produces the batch file to be sent to all the Italian citizens having the Immuni app. Are considered just the foreign keys marked as "Italian" (keys coming from foreign devices that have been in Italy).

Instead, _batch_eu() placed in the process_uploads_eu task, produces the Batch File to be downloaded by the Immuni users that have set the country of interest. These are foreign keys not marked as "Italian" that must be download only by the "Italian travellers".